packages/liveblocks-react-tiptap/src/types.ts

```diff
@@ -0,0 +1,36 @@
+import type { TextSelection } from "@tiptap/pm/state";
+import { PluginKey } from "@tiptap/pm/state";
+import type { DecorationSet } from "@tiptap/pm/view";
+
+export const LIVEBLOCKS_MENTION_KEY = new PluginKey("lb-plugin-mention");
+export const LIVEBLOCKS_MENTION_PASTE_KEY = new PluginKey(
+  "lb-plugin-mention-paste"
+);
+export const LIVEBLOCKS_MENTION_NOTIFIER_KEY = new PluginKey(
+  "lb-plugin-mention-notify"
+);
+export const LIVEBLOCKS_MENTION_TYPE = "liveblocksMention";
```

Comment:
The node type used in Lexical is `lb-mention` and `lb-thread-mark`. Do you see any potential issues with the inconsistency? How do you feel about keeping the names the same?

Reply:
I don't see any issue as the two products won't ever mingle, their data is exclusive of each other.